language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/command/docs/DocumentationGenerator.java

```diff
@@ -171,6 +179,42 @@ private static DocAttachmentInfo generateServiceDocumentation(ServiceDeclaration
         return new DocAttachmentInfo(desc, docStart, getPadding(serviceDeclrNode, syntaxTree));
     }
 
+    /**
+     * Generate documentation for constant declaration node.
+     *
+     * @param constantDeclarationNode constant declaration node
+     * @param syntaxTree       syntaxTree {@link SyntaxTree}
+     * @return generated doc attachment
+     */
+    private static DocAttachmentInfo generateConstantDocumentation(ConstantDeclarationNode constantDeclarationNode,
+                                                                    SyntaxTree syntaxTree) {
+        Optional<MetadataNode> metadata = constantDeclarationNode.metadata();
+        Position docStart = PositionUtil.toRange(constantDeclarationNode.lineRange()).getStart();
+        if (metadata.isPresent() && !metadata.get().annotations().isEmpty()) {
```

Comment:
Do we need to consider whether annotations are present separately? Because the `constantDeclarationNode.lineRange()` should include the metadata as well. Same for the `EnumDeclarationNode` as well.

Reply:
Fixed it and modified the related code snippets.